main.html

```diff
@@ -110,6 +110,18 @@
     <button id="fire">fire</button>
   </div>
 
+  <div class="splash-screen" id="splash-screen">
+    <h1 class="title">Please wait while your app loads</h1>
+    <div class="vertical-flex-container">
+      <div class="flex-buffer"></div>
+      <div>
+        <progress></progress>
+      </div>
+      <div class="flex-buffer"></div>
+    </div>
+    <p class="text">Powered by j2me.js</p>
```

Comment:
I wouldn't display this to users, as it isn't understandable nor useful to them.